modules/social_features/social_group/social_group.api.php

```diff
@@ -51,18 +51,36 @@ function hook_social_group_default_visibility_alter(&$visibility, $group_type_id
   }
 }
 
+/**
+ * Provide a method to alter the allowed visibility used for a group type.
```

Comment:
Can we clarify: is this about the visibility of the group or is this about the content visibility of content posted in a group?

Reply:
I think it is clear from the end of the sentences (group type).

Reply:
I understand that the function is called for a specific group type. But I still don't understand which visibility it alters.

Does it alter
- The visibility of the content in groups with that group type
- The visibility of the groups that are that group type

Reply:
This question is still open.